### PR TITLE
More psm rules

### PR DESCRIPTION
**Pull request by oldchili**

### CODE REVIEW

File: certora/DssLitePsm.spec [hunk 1]
```diff
@@ -1060,3 +1060,91 @@ rule assetsGreaterOrEqualArt(method f) {
...
+
+    sellGem(e, usr, gemAmt);
+
+    storage afterSell = lastStorage;
+
+    sellGemNoFee(e, usr, gemAmt) at initial;
+
+    storage afterSellNoFee = lastStorage;
+
+    assert afterSell[currentContract] == afterSellNoFee[currentContract], "currentContract storage different";
```
Comment (sunbreak1211) [inline certora/DssLitePsm.spec:1084]: In general I was referring `currentContract` as `psm` for errror messages

File: certora/DssLitePsm.spec [hunk 2]
```diff
@@ -1060,3 +1060,91 @@ rule assetsGreaterOrEqualArt(method f) {
     // assert invariant holds after
     assert daiBalanceOfPsmAfter + gemBalanceOfPocketAfter * to18ConversionFactor >= vatUrnPsmArtAfter;
 }
+
+// sellGem with tin as 0 has same effects as sellGemNoFee
+rule sellGemEquivalence() {
+    env e;
+
+    address usr;
+    uint256 gemAmt;
```
Comment (sunbreak1211) [inline certora/DssLitePsm.spec:1069]: For consistency with the general rules I'd call them: `anyAddr` and `anyUint256`

Comment (oldchili) [reply certora/DssLitePsm.spec:1069]: Not sure I understand, in this case this variables are actually used in the rule.
I think that `any` notation is for addresses that are not mentioned directly and we just want to make sure didn't change.

Comment (oldchili) [reply certora/DssLitePsm.spec:1069]: done

**Pull request merged**